### PR TITLE
Shield TV-CVEs -Jan 2022  - (CVE-2021-34401 thru CVE-2021-34406)

### PR DESCRIPTION
**Pull request by PSIRT-NVIDIA**
CVE-2021-34401 thru CVE-2021-34406

### CODE REVIEW

File: 2021/34xxx/CVE-2021-34402.json [hunk 1]
```diff
@@ -1,18 +1,70 @@
 {
```
Comment (cve-team) [inline 2021/34xxx/CVE-2021-34402.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34402.json:1]: ('Invalid control character at: line 37 column 28 (char 1101)',)

File: 2021/34xxx/CVE-2021-34406.json [hunk 2]
```diff
@@ -1,18 +1,70 @@
 {
```
Comment (cve-team) [inline 2021/34xxx/CVE-2021-34406.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34406.json:1]: ('Invalid control character at: line 37 column 27 (char 1100)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by PSIRT-NVIDIA** [comment]
Updated the CVE files, re-submitted them

**Pull request merged**